main.tf

```diff
@@ -88,9 +88,19 @@ locals {
         "awslogs-stream-prefix": "nginx"
       }
     },
-    "environment": [],
+    "environment": [
+      {
+        "name": "PORT",
+        "value": "${var.container_port}"
+      }
+    ],
     "mountPoints": [],
-    "volumesFrom": []
+    "volumesFrom": [],
+    "entryPoint": [
+      "bash",
+      "-c",
+      "curl -o helloworld.go https://gist.githubusercontent.com/pjdufour-truss/7bbebdf5caca22e8f226e925e1a635d4/raw/8d085ca9d7fb4bd6724c195552186bf5f6b9d844/helloworld.go && go run helloworld.go"
```

Comment:
I don't super like having to make a network call to get this container to start.  Could we pass in the contents of the file as a variable, have the entrypoint write it to disk and call it?  I don't think that's much better but would remove the network call.